distribution/src/resources/runbit

```diff
@@ -65,6 +65,47 @@ drill_rotate_log ()
     fi
 }
 
+args=( $@ )
+RBARGS=()
```

Comment:
Does `RBARGS` have some meaning? Maybe it better to give clearer naming?
Also please add comment describing that you remove debug string from original args but leave all other args.